src/cljs/rems/fields.cljs

```diff
@@ -73,7 +88,9 @@
         optional (:field/optional opts)
         value (:field/value opts)
         previous-value (:field/previous-value opts)
-        max-length (:field/max-length opts)]
+        max-length (:field/max-length opts)
+        info-text (localized (:field/info-text opts))
+        collapse-aria-label (str "Info about " title)]
```

Comment:
Like I said in another comment, I don't think there needs to be a parameter `collapse-aria-label` however this is now a hard-coded string. How about localizing it?

Reply:
Yes, i wanted to do it, but forgot after the tests were failing, will address that